hack/components/bump_linux_bridge.sh

```diff
@@ -0,0 +1,48 @@
+#!/usr/bin/env bash
+
+set -xe
+
+source hack/components/common_functions.sh
+
+LINUX_BRIDGE_URL=$(cat components.yaml | shyaml get-value components.linux-bridge.url)
+LINUX_BRIDGE_COMMIT=$(cat components.yaml | shyaml get-value components.linux-bridge.commit)
+LINUX_BRIDGE_REPO=$(echo ${LINUX_BRIDGE_URL} | sed 's#https://\(.*\)#\1#')
+LINUX_BRIDGE_PATH=${GOPATH}/src/${LINUX_BRIDGE_REPO}
+
+echo 'Fetch linux-bridge sources'
+fetch_component ${LINUX_BRIDGE_PATH} ${LINUX_BRIDGE_URL} ${LINUX_BRIDGE_COMMIT}
+
+echo 'Get linux-bridge tag'
+LINUX_BRIDGE_TAG=$(get_component_tag ${LINUX_BRIDGE_PATH})
+
+echo 'Build container image with linux-bridge binaries'
+LINUX_BRIDGE_IMAGE=quay.io/kubevirt/cni-default-plugins
+LINUX_BRIDGE_IMAGE_TAGGED=${LINUX_BRIDGE_IMAGE}:${LINUX_BRIDGE_TAG}
+(
+    cd ${LINUX_BRIDGE_PATH}
+    cat <<EOF > Dockerfile
+FROM golang:1.13 AS builder
```

Comment:
Building and pushing linux-bridge at CNAO feels strange, how were we doing it? maybe it deserve separate repo.

Reply:
We were doing it manually until now.

Separate repository would be a pain to maintain. I'd prefer to keep it here for our own sanity.

I agree that it is weird to build it in the same script we are bumping it. Would `build_linux_bridge.sh` make more sense? I would still call it from this bump script though.

We are consuming U/S components. For some we need to build manifests, for other we need to build whole images.

Reply:
How come there is no built images for CNI plugins like other components?
Should we raise it in containernetworking/plugin repository?

Reply:
Images as a delivery mechanism is a Kubernetes thing. containernetworking/plugins is a generic CNI project that can be used locally with podman etc.